google/resource-snippets/tpu-v1alpha1/tpu.yaml

```diff
@@ -0,0 +1,16 @@
+imports:
+- path: tpu.py
+
+resources:
+- name: tpu
+  type: tpu.py
+  properties:
+    zone: us-central1-b
```

Comment:
For consistency with the other examples, can we change this to be `ZONE_TO_RUN`? Or does this need to explicitly be in `us-central1-b`?

Reply:
Changed it to `ZONE_TO_RUN`